src/main/resources/translations/Messages_fr.properties

```diff
@@ -0,0 +1,163 @@
+# Translators: Please only translate text. Don't change the prefix, colours or formatting.
+# Translators should also supply aliases that match translated versions of the commands
+
+# Minecraft colours may be used in this file for some messages
+# You may use an and-code for these - that is, colour codes beginning with the ampersand "&"
+# Any tokens "%s" are required and will be replaced by the plugin with context-sensitive information - do not remove these
+
+### GENERAL MESSAGES ###
+
+# This is put on the start of some messages
+# Please put a default colour at the end, it's all too easy to forget colours in messages
+message_prefix = &d[&6ArchBlock&d]&f
+
+command_no_permission = &cVous n'avez pas la permission d'utiliser cette commande.
+ownership_blocks_changed = &9Le propriétaire de &c%s&9 blocs a été changé.
+player_only = Cette commande ne peut être utilisée que par un joueur.
+player_or_console_only = Cette commande ne peut être utilisée que par un joueur ou la console.
+task_already_running = &cIl y a déjà une tâche en cours. Veuillez attendre qu'elle finisse.
+unable_to_update_block_owner = Impossible de réattribuer le bloc %s(%s, %s, %s): %s
+unknown_player = &cJoueur inconnu : &b%s
+unknown_world = &cMonde inconnu : &b%s
+
+### COMMAND ALIASES ###
+# Command aliases may be split with a semicolon ";"
+# You can leave any of these commented out if you don't want to use them
+# Translators should provide localized command names as aliases
+
+alias_disown = dp;disownp;
+alias_disownworld = dw
+alias_friend = fr;f
+alias_friends = fs
+alias_setowner = so
+alias_transferblocks = tb;transfer;transferb
+alias_transferplayer = transferp
+alias_unfriend = ufr;uf
+
+### MAIN PLUGIN MESSAGES ###
+
+plugin_disabled_config = Le plugin est désactivé dans la configuration. Modifiez-là ou il ne fera rien !
+plugin_loaded_players = Chargé ! %s joueurs trouvés.
```

Comment:
I don't 100% understand French semantics - Should there be a space before that `!`?